keyboards/deemen17/de80/readme.md

```diff
@@ -0,0 +1,27 @@
+# DE80
+
+![deemen17/de80](https://i.imgur.com/oZvFM0G.png)
```

Comment:
This image shows an encoder, but this PR has no config for one?

Reply:
Thanks for catching that. I'll add that in now.